samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -69,6 +73,12 @@ class BrowserFragment : Fragment(), BackHandler, DownloadDialogListener {
         downloadsFeature.onNeedToRequestPermissions = { _, _ ->
             requestPermissions(arrayOf(WRITE_EXTERNAL_STORAGE), PERMISSION_WRITE_STORAGE_REQUEST)
         }
+
+        // Accept all DRM content
+        object : SelectionAwareSessionObserver(components.sessionManager) {
```

Comment:
Should this be a member class variable and call `observeSelected()` method on `onStop()`?